charts/ocis/values.yaml

```diff
@@ -1228,6 +1228,9 @@ services:
       sha: ""
       # -- Image pull policy
       pullPolicy:
+    # -- Maximum number of concurrent go-routines. Higher values can potentially get work done faster but will also cause 
+    # more load on the system. Values of 0 or below will be ignored and the default value will be used.
+    maxConcurrency: "1"
```

Comment:
frontend has this setting already in 

https://github.com/owncloud/ocis-charts/blob/fcbfec935673da46bc8e617366e2c97fd6b6e629/charts/ocis/values.yaml#L1187-L1188

Reply:
Thanks. Overlooked that. Question is just if we should somehow "unify" the settings. In frontend the old setting is named `concurrency`. In the existing `storageusers` the setting is named `maxConcurrency`. I'm also not sure if that should stay that organised. There is only this specific setting below `events.consumer` and also `events` has only `consumer`. I'm not a big fan of having only one entry per level.

Reply:
For the frontend and events service it's really about how many event consumers should be spawned. That is probably different to the other concurrency settings.